lib/haml_lint/reporter/disabled_config_reporter.rb

```diff
@@ -65,7 +65,7 @@ def finished_file(file, lints)
 
       if lints.any?
         lints.each do |lint|
-          linters_with_lints[lint.linter.name] << lint.filename
+          linters_with_lints[lint.linter.name] |= [lint.filename]
```

Comment:
This looks wrong to me. Won't it mean that only one filename is ever added per lint type?